### PR TITLE
Mgid RTD Module : add new RTD module

### PR DESCRIPTION
**Pull request by velichkin**
## Type of change
- [x] Feature

## Description of change

Mgid Real-time Data Module

Module Name: Mgid Rtd provider
Module Type: Rtd Provider
Maintainer: [prebid@mgid.com](mailto:prebid@mgid.com)

- [x] official adapter submission

### CODE REVIEW

File: integrationExamples/gpt/mgidRtdProvider_example.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+<script>
+      window.googletag = window.googletag || { cmd: [] };
+      googletag.cmd.push(function () {
+        googletag.defineSlot('/46380388/herbeauty_prebid', [[320, 100], [950, 90], [1024, 768], [120, 600], [300, 75], [970, 66], [300, 50], [300, 31], [320, 480], [750, 200], 'fluid', [250, 250], [250, 360], [120, 90], [1, 1], [292, 30], [300, 100], [768, 1024], [88, 31], [200, 200], [300, 600], [300, 1050], [960, 90], [125, 125], [180, 150], [970, 250], [168, 28], [980, 120], [120, 20], [120, 240], [480, 320], [468, 60], [120, 30], [120, 60], [750, 300], [930, 180], [728, 90], [168, 42], [980, 90], [320, 50], [750, 100], [200, 446], [336, 280], [300, 250], [580, 400], [970, 90], [160, 600]], 'div-gpt-ad-1660208650016-0').addService(googletag.pubads());
+        googletag.pubads().enableSingleRequest();
+        googletag.enableServices();
+      });
+    </script>
+<link rel="icon" type="image/png" href="/favicon.png">
+<script async src="//www.googletagservices.com/tag/js/gpt.js"></script>
```
Comment (patmmccann) [inline integrationExamples/gpt/mgidRtdProvider_example.html:14]: Bad location

Comment (patmmccann) [reply integrationExamples/gpt/mgidRtdProvider_example.html:14]: Line 4 above is correct

Comment (velichkin) [reply integrationExamples/gpt/mgidRtdProvider_example.html]: Fixed. Removed line 14.

**Review by Fawke — CHANGES_REQUESTED** [review]
Hi @velichkin,

Your code changes are looking good. Please also add the module in the submodules.json. Take a look at an example [here](https://github.com/prebid/Prebid.js/pull/9135/files#diff-72810593ba1651aea8aecef814185c360f7f2b0c764367bd9c519611b2a63a3c)

And, we would need you to add more documentation and description in the md file. You also need to create a docs PR in this repo - https://github.com/prebid/prebid.github.io

**Comment by velichkin** [comment]
@Fawke I've added module to .submodules.json and made changes in the md file. Also, here is request to docs repo - https://github.com/prebid/prebid.github.io/pull/4081

**Review by Fawke — APPROVED** [review]

**Pull request merged**